### DISCUSSION TITLE
source code for app store sample app

### DISCUSSION THREAD

**Discussion by vigouredelaruse** [CommunityToolkit/Maui#715] [Q&A] [answered]
i apologize for the google search fail here but

the app store sample app ui is a tour de force of things i want to use in more or less exactly the same way

but i cloned this repo and ran the sample solution from [here](https://github.com/CommunityToolkit/Maui/tree/main/samples) and it doesn't look anything like the app store ui

is that code available?
thanks

**Comment by bijington** [comment, marked as answer]
What App Store app are you referring to as we haven't published one. I wonder if it is perhaps for the Windows Community Toolkit?

**Comment by vigouredelaruse** [comment]
indeed. and further i wonder if perhaps i got my clone llink clipboard stack shuffled somehow - thanks

**Comment by CliffAgius** [comment]
Closed as answered
